src/docs/asciidoc/reference/manage/environment-intro.adoc

```diff
@@ -1,3 +1,10 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+Your *application* may be deployed and executed in separate zones called *environments*. Each environment is a particular zone that you can organize as you wish. For example, you can have one environment based on your application lifecycle: `prod`, `pre-prod`, `dev`.
```

Comment:
We talk about lifecycle, I think we should order it in lifecycle order to be consistent: `dev`, `pre-prod`, `prod`